docs/guides/verify-steth-on-optimism-upgrade-manual.md

```diff
@@ -0,0 +1,186 @@
+# stETH on Optimism parameters for Mainnet
+
+## Deployment scope
+
+The full list of the contracts in scope is provided in the Lido Multichain section for [Optimism](https://docs.lido.fi/deployed-contracts/#optimism).
+
+## Ethereum part
+
+### TokenRateNotifier
+
+:::info
+A standalone (non-proxy) contract proposed to plug to the protocol as a new [token rebase receiver](/contracts/lido-locator#posttokenrebasereceiver).
+
+Deployed address: [`0xe6793B9e4FbA7DE0ee833F9D02bba7DB5EB27823`](https://etherscan.io/address/0xe6793B9e4FbA7DE0ee833F9D02bba7DB5EB27823)
```

Comment:
I'm not sure duplicating the address here is better than refer to a single source of truth https://docs.lido.fi/deployed-contracts/#optimism

Reply:
I see your point, will probably remove them :+1: